source/Halibut/Exceptions/RequestCancelledException.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace Halibut.Exceptions
```

Comment:
These exceptions are used to determine where cancellation occurred. Needed in TentacleClient